storage-operator/pkg/controller/volume/volume_controller.go

```diff
@@ -88,85 +488,214 @@ type ReconcileVolume struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileVolume) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Name", request.Name)
-	reqLogger.Info("Reconciling Volume")
+	reqLogger.Info("reconciling volume: START")
+	defer reqLogger.Info("reconciling volume: STOP")
 
-	// Fetch the Volume instance
-	instance := &storagev1alpha1.Volume{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
I'd use `context.WithCancel(context.TODO)` here, because we'd *like* operator-sdk to provide a context, rather than use actively 'choosing' `context.Background` as the parent.

Reply:
Done.